README.md

```diff
@@ -24,6 +24,8 @@ Where opts is an object with the following properties (all optional):
 * **port** - the desired port for the server to use. Defaults to `8000`.
 * **dir** - the directory to launch from (where it should look for a Cordova project). Defaults to cwd.
 * **simhostui** - the directory containing the UI specific files of the simulation host. Defaults to the bundled simulation host files, found in `src/sim-host/ui`.
+* **nolivereload** - A boolean. Set to true to disable live reload. Defaults to false.
+* **forceprepare** - A boolean. Set to true to force a `cordova prepare` whenever a file changes during live reload. If this is false, the server attempts to simply copy the changed file to the platform rather than doing a `cordova prepare`. Ignored if live reload is disabled. Defaults to false.
```

Comment:
Yes, you're right @guillaumejenkins, after making my comment I realize that it will be good for programmatic API but not so good for the CLI. But it was just a minor comment, as it is right now is good!